modules/catalog-magento-px/mappers/ToCategory.ext.ts

```diff
@@ -1,14 +1,8 @@
 import MAGENTO_URL from '#ioc/config/MAGENTO_URL'
+import Extension from '#ioc/types/base/Extension'
+import ToCategoryType from '#ioc/types/catalog-magento-px/ToCategoryType'
 
-interface Extension<Ext = Record<string, never>> {
-  <T extends (...arg: any) => any>(useProduct: T): (...arg: any) => ReturnType<T> & Ext
-}
-
-interface ToCategory {
```

Comment:
This interface should not be extracted to types concept because its local only to this extension.